ansible/roles/test/tasks/lag_2.yml

```diff
@@ -72,9 +72,14 @@
 - name: test each lag interface minimum links and rate
   include: single_lag_test.yml
   with_items: lag_facts.names
-  when: test_minlink|bool == true
+  when: lag_facts.lags[item]['po_config']['runner']['min_ports'] is defined
```

Comment:
I think you should keep the test_minlink here. The test could be disabled when test_minlink is equal to false.

Reply:
OK. got your point.  have reverted this change caused by merge. Thanks